File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/HaltStagedRolloutTask.java
package com.dipien.google.play.api.task;

import com.dipien.google.play.api.App;
import com.dipien.google.play.api.PublishingService;

public class HaltStagedRolloutTask extends BaseTask {
	
	public HaltStagedRolloutTask() {
		setDescription("Halt the current staged rollout");
	}
	
	@Override
	protected void onExecute(App app) {
		new PublishingService().haltStagedRollout(app);
	}
	
}

File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/HaltStagedRolloutTask.kt
package com.dipien.google.play.api.task

import com.dipien.google.play.api.App
import com.dipien.google.play.api.PublishingService

open class HaltStagedRolloutTask : BaseTask() {
    override fun onExecute(app: App) {
        PublishingService().haltStagedRollout(app)
    }

    init {
        description = "Halt the current staged rollout"
    }
}
